File: java/grammer/14_counter.java
import java.util.*;

class GN14Counter {
    public static void main(String[] args) {
        String text = "banana";
        Map<Character, Integer> counter = new HashMap<>();
        for (char c : text.toCharArray()) counter.put(c, counter.getOrDefault(c, 0) + 1);
        System.out.println(counter);
    }
}

File: python/grammer/14_counter.py
"""14) Counter: 원소 빈도 계산."""
from collections import Counter

text = "banana"
counter = Counter(text)

print("문자열:", text)
print("빈도수 dict 형태:", dict(counter))
print("가장 많이 나온 문자 1개:", counter.most_common(1))
